en/admin/installation/installation.md

```diff
@@ -9,6 +9,12 @@ git clone https://github.com/wallabag/wallabag.git
 cd wallabag && make install
 ```
 
+If you want to use LDAP, replace last command with
+```bash
+cd wallabag && make install LDAP_ENABLED=true
+```
```

Comment:
@nicosomb Ok, this seemed dead simple to me, but if it's complicated let's give up. I already spent two weeks on that and I'm not willing to spend more time on it, I’ll just cherry pick on my instance until someone finds the right way to do it.